spec/features/sign_in_out_spec.rb

```diff
@@ -26,7 +26,8 @@
           click_button 'Sign in'
         end
 
-        expect(page).to have_content(@user.username || @user.email)
+        expect(current_url) == '/users/:id'
+        expect(page).to have_content(@user.username)
```

Comment:
Out of curiosity: did you write the expectations before or after implementing the routing scheme and the view?